models/nexusmutual/ethereum/nexusmutual_ethereum_schema.yml

```diff
@@ -146,3 +146,47 @@ models:
       - &eth_egress
         name: egress ethereum
         description: "Sum of outgoing transctions over the time covered by day"
+
+  - name: nexusmutual_ethereum_capital_pool_erc20_transactions
+    meta:
+      blockchain: ethereum
+      project: nexusmutual
+      contributors: [guyeth]
+    config:
+      tags: ['ethereum','nexusmutual','erc20']
+    description: >
+        ERC20 transactions from and to the capital pool
+    tests:
```

Comment:
can you add a unique combo of columns test on columns from [here](https://github.com/duneanalytics/spellbook/blob/main/models/clipper/clipper_trades.sql)? there are examples in a variety of spell schema files